crates/pixi_manifest/src/toml/pyproject.rs

```diff
@@ -25,6 +26,19 @@ pub struct PyProjectToml {
     pub dependency_groups: Option<Spanned<TomlDependencyGroups>>,
 }
 
+impl PyProjectToml {
+    pub fn into_inner(self) -> pyproject_toml::PyProjectToml {
```

Comment:
Maybe this should because `into_concrete_pyproject` or something because for me `into_inner` implies we are wrapping that type like an `Arc<T>` for example and this is not the case.

Reply:
The other similar methods in that file that do the same are called `into_inner`. So keeping it consistent makes sense to me? But yeah, maybe not the best name :)